honghyojeong/likelion-assignment-crud/src/main/java/com/likelion/likelionassignmentcrud/reservation/application/ReservationService.java

```diff
@@ -0,0 +1,44 @@
+package com.likelion.likelionassignmentcrud.reservation.application;
+
+import com.likelion.likelionassignmentcrud.customer.domain.Customer;
+import com.likelion.likelionassignmentcrud.customer.domain.repository.CustomerRepository;
+import com.likelion.likelionassignmentcrud.reservation.api.dto.request.ReservationSaveRequestDto;
+import com.likelion.likelionassignmentcrud.reservation.api.dto.response.ReservationInfoResponseDto;
+import com.likelion.likelionassignmentcrud.reservation.api.dto.response.ReservationListResponseDto;
+import com.likelion.likelionassignmentcrud.reservation.domain.Reservation;
+import com.likelion.likelionassignmentcrud.reservation.domain.repository.ReservationRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly=true)
+public class ReservationService {
+    private final CustomerRepository customerRepository;
+    private final ReservationRepository reservationRepository;
+
+    @Transactional
+    public void reservationSave(ReservationSaveRequestDto reservationSaveRequestDto) {
+        Customer customer = customerRepository.findById(reservationSaveRequestDto.customerId()).orElseThrow(IllegalArgumentException::new);
```

Comment:
코드는 잘 작성해주셨습니다! 다만, 가독성을 위해 적절한 줄바꿈과 들여쓰기도 중요합니다. 예를 들어, 메서드 체인이 길어지면 각 단계별로 줄바꿈을 해주면 읽기 편해집니다. 
```suggestion
        Customer customer = customerRepository.findById(reservationSaveRequestDto.customerId())
        .orElseThrow(IllegalArgumentException::new);
```
이렇게 작성하면 코드가 한눈에 들어와서 리팩토링 하기 더 좋아요! 다른 부분들도 수정해주시면 좋을 것 같습니다~